src/data.ts

```diff
@@ -27,14 +27,23 @@ const TOKEN_TYPES = Object.freeze({
 
 // CONTRACT VERSIONS
 // TODO: rename CONTRACT to VAULT
-const LATEST_STABLE_CONTRACT_VERSION = 'v4.4'
+const LATEST_STABLE_CONTRACT_VERSION = 'v4.3'
 const LATEST_EXPERIMENTAL_CONTRACT_VERSION = 'v4.4'
-const FALLBACK_CONTRACT_VERSION = 'v4.3'
+const FALLBACK_CONTRACT_VERSION = 'v4.2'
 const LATEST_STABLE_ROUTER_VERSION = 'Rv4'
-const LATEST_EXPERIMENTAL_ROUTER_VERSION = 'Rv4'
-const LATEST_STABLE_BATCHER_VERSION = 'Bv4.4'
+const LATEST_EXPERIMENTAL_ROUTER_VERSION = 'Rv4.2'
+const LATEST_STABLE_BATCHER_VERSION = 'Bv4'
 const LATEST_EXPERIMENTAL_BATCHER_VERSION = 'Bv4.4'
 
+// Set of arrays for features only available on certain versions.
+// !! IMPORTANT !!: Update these arrays when creating new contracts
+// !! IMPORTANT !!: When creating a new array, make sure it starts with CA (standing for Contract Array).
+const CA_V4 = ['v4', 'v4.2', 'v4.3', 'v4.4']
+const CA_V4_2_ANDUP = ['v4.2', 'v4.3', 'v4.4']
+const CA_SUPPORTS_MFA = ['v4.4']
+const CA_SUPPORTS_MFA_BATCHER = ['Bv4.4']
+const CA_SUPPORTS_MFA_ROUTER = ['Rv4.2']
```

Comment:
Nit: I think 'CA_SUPPORTS_MFA' is slightly awkward. It'd read ContractArraySupportsMFA, which syntax wise is a bit confusing.

Also, CA_SUPPORTS_MFA_BATCHER  imo is confusing too, as the contract type (e.g. vault router or batcher) should be next to CA i think

What do you think about these schemes:
- CONTRACTS_SUPPORTING_MFA
- VAULT_CONTRACTS_SUPPORTING_MFA
- VAULT_CONTRACTS_WITH_MFA
- BATCHER_CONTRACTS_WITH_MFA
- VAULT_CONTRACTS_V4_2_ANDUP

I think "with" > supporting > supports